41_snapshots-btrfs

```diff
@@ -6,30 +6,36 @@
 # BTC donation address: 1Lbvz244WA8xbpHek9W2Y12cakM6rDe5Rt
 # Github: https://github.com/Antynea/grub-btrfs
 #
-# Purpose: Include btrfs snapshots at boot options (grub-menu).
+# Purpose:
+#   Improves Grub by adding "btrfs snapshots" to the Grub menu.
+#   You can start your system on a "snapshot" from the Grub menu.
+#   Supports manual snapshots, snapper ...
+#   Warning : it isn't recommended to start on read-only snapshot
 #
 # What this script does:
-# Simple rollback using snapshots you made previously.
 # - Automatically List snapshots existing on root partition (btrfs).
 # - Automatically Detect if "/boot" is in separate partition.
 # - Automatically Detect kernel, initramfs and intel microcode in "/boot" directory on snapshots.
-# - Automatically Create corresponding "menuentry" in grub.cfg , which ensures a very easy rollback.
+# - Automatically Create corresponding "menuentry" in grub.cfg
 # - Automatically detect snapper and use snapper's snapshot description if available.
+# - Automatically generate grub.cfg if you use the provided systemd service.
 #
-# How to customize it:
-# - Refer to config for the list of available options and their default values.
-# - Place your configurations to /etc/default/grub-btrfs/config.
+# Installation:
+# - Copy 41_snapshots-btrfs to /etc/grub.d/. (make sure the permissions are 755)
+# - Copy config to /etc/default/grub-btrfs/. (make sure the permissions are 644)
+# - Copy grub-btrfs.path to /etc/systemd/system/ or /lib/systemd/system/. (make sure the permissions are 644)
+# - Copy grub-btrfs.service to /etc/systemd/system/ or /lib/systemd/system/. (make sure the permissions are 644)
+# - Generate your Grub menu after installation for the changes to take effect. (on Arch linux use grub-mkconfig -o /boot/grub/grub.cfg)
```

Comment:
Are you sure you want to describe it in text? If Makefile changes, need to remember to update this... Maybe just say:

```
Run `make install` or look into Makefile for instructions on where to put each file.
```

Reply:
you are right :)

Reply:
Don't forget to do the same in README 😉 Otherwise looks very good!